l3build.lua

```diff
@@ -115,45 +115,44 @@ unpackdir     = escapepath(unpackdir)
 -- Tidy up the epoch setting
 -- Force an epoch if set at the command line
 -- Must be done after loading variables, etc.
-if options["epoch"] then
-  epoch           = options["epoch"]
+if Opts.epoch then
+  epoch           = Opts.epoch
   forcecheckepoch = true
   forcedocepoch   = true
 end
 normalise_epoch()
 
 -- Sanity check
-check_engines()
+A.check_engines(Opts, checkengines)
 
 --
 -- Deal with multiple configs for tests
 --
 
 -- When we have specific files to deal with, only use explicit configs
 -- (or just the std one)
-if options["names"] then
-  checkconfigs = options["config"] or {stdconfig}
+if Opts.names then
+  checkconfigs = Opts.config or {stdconfig} -- What is stdconfig?
```

Comment:
`stdconfig` is defined (like most other config data) in `l3build-variables`: it's the name of the main `build.lua` file, which of course could be something else ...

Reply:
I can't find any `stdconfig` out of `l3build.lua`.

Reply:
> I can't find any `stdconfig` out of `l3build.lua`.

Ah, right: this is one I think we renamed, so it could go yes.